docs/sql/json.md

```diff
@@ -55,25 +47,13 @@ the JSON `null` value.  An important difference is that two `VARIANT`
   is subject to the capitalization rules of the SQL dialect, so for correct
   operation the field may need to be quoted: `variant."field"`
 
-The runtime types do not match exactly the compile-time types:
```

Comment:
I think this is important and should not be deleted

Reply:
It sounds like an internal implementation detail the way it's documented. If it's important, let's explain what's actually important to the user.

Reply:
It's important because of two reasons : the types can be used using type of in a case statement to achieve the effect of rust match. Second, it tells you which casts will work

Reply:
Will add an example using case

Reply:
Let's do a separate PR.